syntax = "proto3";

option java_multiple_files = true;
option java_package = "map-reduce";
option java_outer_classname = "MapReduce";
option objc_class_prefix = "DSC";

package mapReduce;

service Reducer {
    rpc reduce(ReduceRequest) returns (ReduceResponse) {}
}

message ReduceRequest {
    map <string, string> inputpaths = 3;
}

message ReduceResponse {
    string status = 2;
}